src/polar/Radar.tsx

```diff
@@ -110,11 +110,12 @@ export class Radar extends PureComponent<Props, State> {
     const { cx, cy } = angleAxis;
     let isRange = false;
     const points: RadarPoint[] = [];
+    const angleBandSize = angleAxis.type !== 'number' ? bandSize ?? 0 : 0;
```

Comment:
what path does this go through when axis type is number
https://github.com/recharts/recharts/blob/master/src/util/ChartUtils.ts#L1271

Is there anything we can do in that function to guard against this?

Reply:
I have modified the Radar to minimize side effects,
It seems appropriate to make the changes at this line